src/cascadia/TerminalApp/App.cpp

```diff
@@ -1307,8 +1315,15 @@ namespace winrt::TerminalApp::implementation
                                             _settings->GlobalSettings().GetDefaultProfile();
         const auto controlSettings = _settings->MakeSettings(realGuid);
 
-        // Create a Conhost connection based on the values in our settings object.
-        TerminalConnection::ITerminalConnection controlConnection = TerminalConnection::ConhostConnection(controlSettings.Commandline(), controlSettings.StartingDirectory(), 30, 80, winrt::guid());
+        TerminalConnection::ITerminalConnection controlConnection{ nullptr };
+        if (controlSettings.Commandline() == L"Azure")
+        {
+            controlConnection = TerminalConnection::AzureConnection(30, 80);

```

Comment:
If 30, 80 is used here and below, it should probably be used from a shared constant variable, not defined twice.

Reply:
Ah I was just following the way the Conhost connection is initialized. Will change all of them to be shared variables.